### ISSUE TITLE
remoting.context option was removed in version 3.0

### ISSUE THREAD

**Issue by dcversus** [strongloop/loopback#2660]
Hi!
Today, after npm i in my app i get error: (using loopback ^3.0.0-alpha.3)
```
Unhandled error for request GET /myRestRoute: Error: remoting.context option was removed in version 3.0. See https://docs.strongloop.com/display/APIC/Using%20current%20context for more details
```

Then, i try create empty app (for testing) and get same error. My steps:

```
$ npm i -g strongloop
$ slc loopback
(select -  api-server (A LoopBack API server with local User auth))
$ node .
```
in browser http://localhost:3000/api/Users
and get: 
```
Unhandled error for request GET /api/Users: Error: remoting.context option was removed in version 3.0. See https://docs.strongloop.com/display/APIC/Using%20current%20context for more details.
```
and after refresh
```
Unhandled error for request GET /api/Users: Error: Cannot GET /api/Users
```

Please help! What other information do I need to provide to you may help me?

**Comment by davidcheung** [comment]
Hi @dcversus, thanks for bringing this up,
we are currently waiting for https://github.com/strongloop/loopback-workspace/commit/72dd1fc14ca3c6ea220b4092ada9bb6fa958f5f5 to be published to workspace which would update the scaffolding. 

### Temp solution 
in the mean time you would have to manually remove the option from 
```js 
//config.json remove `context` from remoting object
"context": {
  "enableHttpContext": false
}
```

/cc @bajtos

**Comment by bajtos** [comment]
Released `loopback-workspace@3.33.0`.

**Comment by sebfindling** [comment]
how do i update my project to fix this problem?
